webmails/roundcube/Dockerfile

```diff
@@ -7,9 +7,12 @@ ONBUILD COPY --from=balenalib/rpi-alpine:3.14 /usr/bin/qemu-arm-static /usr/bin/
 FROM ${ARCH}php:7.4-apache as build_other
 
 FROM build_${QEMU}
-
+ARG VERSION
 ENV TZ Etc/UTC
 
+LABEL version=$VERSION
+RUN echo $VERSION >> /version
```

Comment:
I think this should be the last "RUN" command in the Dockerfiles's to allow using unmodified / cached states.

Reply:
Yes. That makes much sense. I will make this change.